HTML/index.html

```diff
@@ -40,19 +47,18 @@ <h6 class="test">THE BEST FOOD BLOG ON THE WEB.</h6>
     </header>
 
     <main class="test">
-
       <article class="test">
         <section class="test">
-          <h1 class="title grid_12 test">Chocolate Pizza</h1>
-          <h5 class="subtitle grid_6 test">POSTED ON 15 DEC 2013 / DESSERTS<h5>
+          <h1 class="title grid_16 test">Chocolate Pizza</h1>
+          <h5 class="subtitle grid_8 test">POSTED ON 15 DEC 2013 / DESSERTS<h5>
           <div class="print-text grid_6 test"><img class="print-icon" src="file:///Users/jkoehnke/Downloads/challenge/print-icon.png" alt="Print"> PRINT </div>
```

Comment:
Make sure the text within your div is inside a 'p' or h* element. It's auto rendering as an h5 in the browser.